pages/phases/phone-assessment-interviews/training.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: article
+permalink: /hiring-phases/phone-assessment-interviews/training/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+sub-phase-order: 015
+title: Training SMEs to Conduct Interviews
+intro: While phone interviews are being scheduled with applicants, facilitate a mandatory two-hour interview training for all SMEs who will conduct the next round of interviews that explains how to conduct structured interviews and assess an applicant’s proficiency with the required competencies based on their responses.
+---
+
+Before you schedule training for the first round of interviews, divide the SMEs into two groups based on which round of interviews they will conduct. Consider how many applicants passed resume review, as well as any scheduling conflicts you may be aware of at this time.
+
+### Setting Up Phone Assessment Interview Training
+
+Send calendar invites to all SMEs who will conduct the next round of interviews for your hiring action. SMEs can participate in the phone interview training in person or remotely using a conference bridge line, depending on the means and norms at your agency.
```

Comment:
clarify that you'll have two trainings - one per SME interview group. explain this is becausae you'll review the questions for that specific interview in depth during the training so you only want the SMEs doing that interview to be present.